### ISSUE TITLE
Bug:  TimeseriesForecaster--Process finished with exit code -1073740791 (0xC0000409)

### ISSUE THREAD

**Issue by Hunter-Bo** [keras-team/autokeras#1749]
when I run docs/py/timeseries_forecaster.py, process finished.

Python Console:

(6549, 12)
(6549,)
INFO:tensorflow:Reloading Oracle from existing project .\time_series_forecaster\oracle.json
2022-07-04 18:22:54.926686: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-07-04 18:22:55.439948: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 12467 MB memory:  -> device: 0, name: Quadro RTX 5000, pci bus id: 0000:17:00.0, compute capability: 7.5
Search: Running Trial #1
Value             |Best Value So Far |Hyperparameter
True              |?                 |timeseries_block_1/rnn_block_1/bidirectional
lstm              |?                 |timeseries_block_1/rnn_block_1/layer_type
2                 |?                 |timeseries_block_1/rnn_block_1/num_layers
0                 |?                 |regression_head_1/dropout
adam              |?                 |optimizer
0.001             |?                 |learning_rate
Epoch 1/10
Process finished with exit code -1073740791 (0xC0000409)

**Setup Details**
Include the details about the versions of:

OS type and version: Win10 1903
Python: 3.8.8
autokeras: 1.0.19
keras-tuner: 1.1.2
scikit-learn: 1.0.2
numpy: 1.22.3
pandas: 1.3.5
tensorflow: 2.8.0
